### ISSUE TITLE
Compilation failed

### ISSUE THREAD

**Issue by maikol-solis** [skranz/glueformula#1]
Hi! 

Trying to install this package give me this output. 

```{r}
Loading required package: glueformula
Downloading GitHub repo skranz/glueformula@master
✔  checking for file ‘/tmp/RtmpqvDqnG/remotes112e68386f2c/skranz-glueformula-a89a968/DESCRIPTION’ ...
─  preparing ‘glueformula’:
✔  checking DESCRIPTION meta-information ...
   Warning: /tmp/RtmpviEwJB/Rbuild1e5069de49cc/glueformula/man/varnames_snippet.Rd:15: unexpected RCODE '(x), ', expecting '{'
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘glueformula_0.1.0.tar.gz’
   
Installing package into ‘/home/msolis/R/x86_64-pc-linux-gnu-library/3.6’
(as ‘lib’ is unspecified)
* installing *source* package ‘glueformula’ ...
** using staged installation
** R
** byte-compile and prepare package for lazy loading
** help
Error : (converted from warning) /tmp/RtmprTItUU/R.INSTALL1e6120da7db/glueformula/man/varnames_snippet.Rd:15: unexpected RCODE '(x), ', expecting '{'
ERROR: installing Rd objects failed for package ‘glueformula’
* removing ‘/home/msolis/R/x86_64-pc-linux-gnu-library/3.6/glueformula’
```

Thanks for the help.

**Comment by maikol-solis** [comment]
I didn't read all through. Run this fixed it 

```
Sys.setenv(R_REMOTES_NO_ERRORS_FROM_WARNINGS="true")
```

**Comment by skranz** [comment]
Thanks a lot for for the heads up. I did not intend this warning, but updated some code and overlooked that warning when testing the package. I added the the line 

`Sys.setenv(R_REMOTES_NO_ERRORS_FROM_WARNINGS="true")`

in the installation instructions, since I know from past experience that such thing can happen and, even worse that sometimes errors are thrown just because not all of the users packages are up-to date.  I updated `glueformula` and at least on my computer installation works again without warning / error.
